### ISSUE TITLE
Square black bullet symbol not accessible

### ISSUE THREAD

**Issue by dajoturner** [ucsc/wcms-design-templates#271]
Len Burns issue. Len tested with an iPhone, and found an issue that may be related to one or more of his other tickets (most likely #269, but possibly also #264 and #69). 
See: http://www-dev.ucsc.edu/about/facts-figures.html

Assigning to Rob and setting to Critical, since this is design related and everywhere.

Len's text:
Testing under IOS finally clarified the source of this problem.  Many
unnumbered lists are using what is being identified as a black square,
presumably for a bullet.  Several problems are resulting from this.  First,
the computer based screen readers do not have this character in their
character dictionaries.  For this reason, it is apparent something is
present, but unclear at a glance what it is.  When using the element by
element navigation in Jaws, Jaws lands on this bullet, presumably because it
is perceived as the first line.  This means that where I would expect to
hear: "Facts and Figures," I hear only silence.  Under IOS it is rendered as
a "black square." So, as I swipe right to read through a list I hear:
"black square"
Facts and Figures
"Black Square"
Achievements
"Black Square"
University News
The IOS version results in a lot of extra speech out, since "black square"
is read prior to each list element.  And an extra swipe per element is also
required.  With a Braille terminal it is more confusing.  In Jaws, the
outcome is mostly confusing.  The involved character is described thus:
" Emoji Gemoji image for :black_small_square
HTML Entity (decimal) &#9642;
HTML Entity (hex) & #x25aa;"
